cfgov/unprocessed/apps/paying-for-college/css/college-costs/financial-item.less

```diff
@@ -89,10 +99,19 @@
     .flex-container {
       display: flex;
 
-      .o-expandable_label {
+      .o-costs-group_label {
         flex: 1;
+        text-align: left;
```

Comment:
What's making the `text-align: left` necessary? This isn't the default?

Reply:
Oh wait, no, `<BUTTON>` centers text, which is why it's there, but there may be a better way to do this...

Reply:
Oh right, from https://github.com/cfpb/design-system/blob/main/packages/cfpb-expandables/src/expandable.less#L55 … I agree. I wonder if there's a better way than override button defaults.